styles.css

```diff
@@ -0,0 +1,115 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f4f5f7;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+}
+
+.header {
+  height: 100px;
+}
+
+.header-container {
+  height: 92px;
+  width: 100%;
```

Comment:
- [ ] Please be sure to stick to the Figma template as much as possible. In your design, there is a gray spacing between the header and the headline that doesn't appear on the Figma design.

Your design 👇🏽 

![Screenshot from 2023-07-08 14-10-40](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/10439283/74c22f12-28cd-4282-938f-30b27927b8df)

Design from Figma 👇🏽 

![Screenshot from 2023-07-08 14-13-33](https://github.com/Eugeneiyukhun/Setup_and_Mobile_First/assets/10439283/4a5f8cd0-d4a0-4f17-85c9-bd0d61d4eacb)

You might fix that by setting the height in your class **header-container** to `100px` and setting the parent container to auto to prevent _overflow_  
```
.header {
  height: auto;
}
```